content/swiftui/concepts/views/terms/textfield/textfield.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: 'TextField'
+Description: 'It a view that allows user to insert their informations, type memos and edit the content.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`TextField`** is a view that allows user to fill in the field with the relative data. 
+The first parameter refers to the text that will be displayed in a lighter color inside the field. 
```

Comment:
Lines 17 and 18 should be in the syntax section, and should refer back to placeholders in the ```pseudo code block.